csuntechlab/iSTART
# STR-129 -- FE: Create single question multiple choice slide template

# Description
'Write a meaningful description of your 
There exists a template that reflects the single question multiple choice slides.
Should include styling as well as the functionality
Should be mobile responsive
# Testing
'What does the reviewer need to do to test your PR?'
1. Login as `apirl.feldman`
2. Click on any module
3. Make sure that the "continue" button doesn't appear
3. Select anything to see the functionality such as styling that displays the wrong and right answer
4. Click continue to see that the icons and styling disappear and that the continue button itself disappears

# Installation
'Are there any dependencies the reviewer needs to download?'
